### DISCUSSION TITLE
Uses only system theme

### DISCUSSION THREAD

**Discussion by yusadogru** [mozilla-mobile/firefox-ios#9399] [General]
Hi, Firefox supports iOS 13 and above so maybe it uses only system theme 🤔 Why do we need a manual theme feature? I think manual theme increases code complexity and there is applyTheme in everywhere :(

**Comment by adudenamedruby** [comment]
Heya. You're totally right.

Our manual theming system is a holdover from when we supported iOS versions below iOS 13 - we've only dropped support for iOS 11/12 earlier this year. This week at some point, I'll be implementing a new theme system in accordance with our new design system. Moving forward, we will be using that new theme system, and, as we address features or bugs, we will also go backwards and fix the current theming system.

**Reply by yusadogru** [reply]
Nice, I like that :)
